inst/model/nonmem/test-workflow-bbi-models/1.yaml

```diff
@@ -0,0 +1,10 @@
+model_type: nonmem
```

Comment:
This file doesn't looked use by the tests.

Reply:
I think I changed this so when it was copied it would have a value under starred.

Reply:
This whole folder should be deleted (i.e. not tracked). It is created fresh [here](https://github.com/metrumresearchgroup/bbr/blob/develop/tests/testthat/test-workflow-bbi.R#L18-L26) every time and should be deleted [here](https://github.com/metrumresearchgroup/bbr/blob/develop/tests/testthat/test-workflow-bbi.R#L33-L37) but somehow it seems to have gotten stuck and checked in by accident.